python/paddle/device/cuda/__init__.py

```diff
@@ -14,7 +14,9 @@
 
 import paddle
 from paddle.fluid import core
+from paddle.fluid import framework
 from paddle.fluid.wrapped_decorator import signature_safe_contextmanager
+from paddle.device import is_compiled_with_cuda
```

Comment:
I guess this line would cause circular dependencies. Are you sure it is right? If you have checked, please tell me.

My thought on why there are circular dependencies: 
- When users import `paddle.device.cuda`, Python would import `python/paddle/device/cuda/__init__.py` and reach this line `from paddle.device import is_compiled_with_cuda`. 
- Then, Python would try to import `paddle.device`. Inside `python/paddle/device/__init__.py`, there is the code `from . import cuda`. So, Python would import `python/paddle/device/cuda/__init__.py` again. That is why I think the circular dependencies occur.